diff -dur dssi-1.0.0.orig/configure.ac dssi-1.0.0/configure.ac
--- dssi-1.0.0.orig/configure.ac	2009-01-05 02:47:50.000000000 +0100
+++ dssi-1.0.0/configure.ac	2009-12-06 21:50:57.000000000 +0100
@@ -68,14 +68,7 @@
   fi
 else
   AC_MSG_RESULT([$QTDIR])
-  AC_MSG_CHECKING([for Qt library])
-  if test -f $QTDIR/lib/libqt-mt.so -o -f $QTDIR/lib/libqt-mt.a; then
     with_qt=yes
-    AC_MSG_RESULT([in $QTDIR/lib])
-  else
-    with_qt=no
-    AC_MSG_RESULT([no. Perhaps QTDIR is not set correctly?])
-  fi
 fi
 if test "x${with_qt}" = xyes ; then
   if test "x${QTINCDIR}" != x ; then
